.eslintrc.js

```diff
@@ -275,6 +275,7 @@ module.exports = {
       files: ['docs/**/*{.ts,.tsx,.js}'],
       excludedFiles: ['*.d.ts'],
       rules: {
+        'material-ui/no-hardcoded-labels': 'off',
```

Comment:
Added this to fix the translation errors, on v8 we are doing line-by-line for some reason. Though it should be easier to just do this on v7

Reply:
I think we agreed to be explicit in that regard. 🤔 
Reference: https://github.com/mui/mui-x/pull/15644#issuecomment-2507523221
IMHO, we should be consistent with solutions for the `master` and `v<VERSION>.x` branches if possible. 🙈